src/roadmapper/milestone.py

```diff
@@ -59,6 +58,38 @@ def draw(self, painter: Painter) -> None:
                 self.diamond_height,
                 self.fill_colour,
             )
+
+        alignment = (
+            self.text_alignment.lower() if self.text_alignment is not None else None
+        )
+
+        if alignment is None or alignment == "centre":
```

Comment:
Should we consider both spellings `centre` and `center`? (This would then also have to be considered in the error message in the last else block.)

Reply:
In my most recent commit the Alignment class handles these two being aliases by using an Enum subclass.